test_definitions.py

```diff
@@ -106,3 +106,31 @@ def test_filename_matches_type_version_fields(definition_file):
     # Do they match what's in the definition?
     assert type == definition_file.definition["_name"]
     assert version == definition_file.definition["_version"]
+
+
+@pytest.mark.parametrize(
+    "definition_file",
+    EVENT_DEFINITIONS,
+    ids=EVENT_DEFINITION_IDS,
+)
+def test_links(definition_file, manifest):
+    # Checking for required link is only valid for event types with recent meta schemas
+    if "draft-04" in definition_file.definition.get("$schema"):
+        assert True
```

Comment:
Nit: How about doing this outside of the test instead? Saying the test passed is a bit misleading since it's actually skipped.

```diff
diff --git a/test_definitions.py b/test_definitions.py
index 89d3e76..a708665 100644
--- a/test_definitions.py
+++ b/test_definitions.py
@@ -38,6 +38,8 @@ EVENT_DEFINITIONS = [
     DefinitionFile(p) for p in DEFINITION_FILES if p.parent.name.endswith("Event")
 ]
 EVENT_DEFINITION_IDS = [d.id for d in EVENT_DEFINITIONS]
+EVENT_DEFINITIONS_DRAFT2020_12 = [d for d in EVENT_DEFINITIONS if "draft-2020-12" in d.definition.get("$schema")]
+EVENT_DEFINITION_DRAFT2020_12_IDS = [d.id for d in EVENT_DEFINITIONS_DRAFT2020_12]
 OTHER_DEFINITIONS = [
     DefinitionFile(p) for p in DEFINITION_FILES if not p.parent.name.endswith("Event")
 ]
@@ -110,8 +112,8 @@ def test_filename_matches_type_version_fields(definition_file):
 
 @pytest.mark.parametrize(
     "definition_file",
-    EVENT_DEFINITIONS,
-    ids=EVENT_DEFINITION_IDS,
+    EVENT_DEFINITIONS_DRAFT2020_12,
+    ids=EVENT_DEFINITION_DRAFT2020_12_IDS,
 )
 def test_links(definition_file, manifest):
     # Checking for required link is only valid for event types with recent meta schemas
```

Reply:
Great suggestion! Will fix.

Reply:
Resolved as the PR is approved by the comment author